application/common/models/User.php

```diff
@@ -145,7 +151,7 @@ private function savePassword($runValidation = true): bool
 
             $this->password_hash = password_hash($this->password, PASSWORD_DEFAULT);
 
-            if (! parent::save($runValidation, ['password_hash'])) {
+            if (! parent::save()) {
```

Comment:
Umm... I'm sorry. I'm not sure which way that "no" means, but since you're planning on adding a test to catch what I was concerned about here (possibly changing other fields through a call to `savePassword()`), I'll assume you've got that taken care of.